### DISCUSSION TITLE
size of TextEntry on infolists dont works

### DISCUSSION THREAD

**Discussion by JayAyAre** [filamentphp/filament#11431] [Help]
### Package

Infolist builder

### Package Version

v3.0.0

### How can we help you?

This is my code and my problem is that I have tried in multiple ways (comments //) to change the font size of the information that is shown to me on the screen, but even so, I cannot change the font size in any way, the only thing is What I want is to increase the size or be able to display the information of my object in some more striking way, but what I really want is to know why the size is not changing.

I tried to do a weight to see if it worked, and yes, it works. The only thing that is giving me problems is the size.

No way seen in the code has worked for me so far.   
    public function infolist(Infolists\Infolist $infolist): Infolists\Infolist
    {
        /* @phpstan-ignore-next-line */
        $plan = $this->record->plan;
        return $infolist
            ->schema([
                Infolists\Components\Section::make('Client Details')
                    ->schema([
                        Infolists\Components\TextEntry::make('clients.name')
                            ->label('Name')
                            ->size(Infolists\Components\TextEntry\TextEntrySize::Small),
                            //->weight(FontWeight::Bold),
                            //->state(BudgetClient::query()->find($record->budget_client_id)?->name)
                            //->size(Infolists\Components\TextEntry\TextEntrySize::Large),
                            //->extraAttributes(['class' => 'big-text-size']),
                        Infolists\Components\TextEntry::make('clients.last_name')
                            ->label('Last name'),
                        Infolists\Components\TextEntry::make('clients.email')
                            ->label('Email')
                            
                            
![image](https://github.com/filamentphp/filament/assets/73116131/68c67823-a732-435b-8822-4df1cb785793)

**Comment by tobiasla78** [comment]
If i also use `->size(Infolists\Components\TextEntry\TextEntrySize::Large),` it will not work but if i use `->size(\Filament\Infolists\Components\TextEntry\TextEntrySize::Large)` it works, maybe it's just a namespace problem. Do you get any error?

**Reply by JayAyAre** [reply]
@tobiasla78 nop, i tried with             
->schema([
                Infolists\Components\Section::make('Client Details')
                    ->schema([
                        Infolists\Components\TextEntry::make('clients.name')
                            ->label('Name')
                            ->extraAttributes(['class' => 'big-input-column-width'])
                            ->size(\Filament\Infolists\Components\TextEntry\TextEntrySize::Large),
                            
                            and still dont works

**Reply by leandrocfe** [reply]
`->size('lg')` should work.
You can also create a custom view to customize what you want

```php
TextEntry::make('clients.name')
    ->view('custom-text')
```

```html
<!-- resources/views/custom-text.blade.php -->
<div class="text-2xl">
    {{ $getState() }}
</div>
```
